_assets/js/netlify/getPreviewLink.js

```diff
@@ -0,0 +1,16 @@
+export default function getPreviewLink(entry) {
+    let link;
+    const card = entry.getIn(['data', 'card']);
+    if (card) {
+        link = card.get('href');
+    } else {
+        link = entry.getIn(['data', 'permalink']);
+    }
+    const status = entry.get('status');
+
+    if (status === '') {
+        return null;
+    }
+
+    return 'https://federalist-1716bf9c-dd79-4d96-8285-6e56dc391b84.app.cloud.gov/preview/usdoj-crt/beta-ada/cms'.concat(link);
+}
```

Comment:
I see three cases to this - below might be a more concise way to express this?
- No status -> null
- Is a card -> card href
- Else -> permalink

Is it safe to assume permalink exists?

```suggestion
const URL = 'https://federalist-1716bf9c-dd79-4d96-8285-6e56dc391b84.app.cloud.gov/preview/usdoj-crt/beta-ada/cms';

export default function getPreviewLink(entry) {
  if (entry.get('status') === '') return null;

  const card = entry.getIn(['data', 'card']);
  if (card) return URL + card.get('href');
    
  const permalink = entry.getIn(['data', 'permalink'])
  if (permalink) return URL + permalink;
  
  console.error('Unable to build pages from entry', entry);
  return null;
```